gcp_variant_transforms/libs/variant_merge/move_to_calls_strategy_test.py

```diff
@@ -225,20 +226,28 @@ def test_get_merged_variants_move_everything_to_calls(self):
   def test_get_merge_keys(self):
     strategy = move_to_calls_strategy.MoveToCallsStrategy(None, None, None)
 
+    empty_string_hash = hashlib.sha256('').hexdigest()
```

Comment:
Consider making a helper method here as well. I think it's ok to use the one in the private lib too.

Reply:
Done!